packages/core/components/progressBar/index.mdx

```diff
@@ -28,3 +58,30 @@ Displays a fractional amount of progress towards some end.
 | borderRadius | string or number | no | `999` | |
 | width | string | no | `100%` | |
 | height | string | no | `0.8rem` | |
+| striped | boolean | no | false | |
+| animated | boolean | no | false | animates the striped variant |
+
+## Theming
+
+### Schema
+```
+{
+  colors: {
+    bg,
+    color,
+  },
+  radii: {
+    borderRadius
+  },
+  heights: {
+    height
+  },
+  styles: {
+    backBar: css``,
+    topBar: css``
+  }
+}
+```
+
+- [Default theme](https://github.com/raster-foundry/blasterjs/tree/master/packages/core/theme/components/progressBar)
+- [Learn more about themeing](#)
```

Comment:
Probably could be removed until we have theming docs

Reply:
This is throughout the docs. I will remove this line in all docs in another pr.